docs/documentation/functions/system-prechecks/Get-VCFSystemPrecheckTask.md

```diff
@@ -24,6 +24,13 @@ Get-VCFSystemPrecheckTask -id 4d661acc-2be6-491d-9256-ba3c78020e5d
 
 This example shows how to retrieve the status of a system level precheck task by unique ID.
 
+### Example 2 
+```powershell
+Get-VCFSystemPrecheckTask -id 4d661acc-2be6-491d-9256-ba3c78020e5d -failedonly
+```

```

Comment:
```suggestion
### Example 2

```powershell
Get-VCFSystemPrecheckTask -id 4d661acc-2be6-491d-9256-ba3c78020e5d -failedonly
```
```

Reply:
I believe this is fixed in the latest commit